app/ts/utils/utilities.ts

```diff
@@ -73,3 +73,11 @@ export function isSameAddress(address1: `0x${ string }` | undefined, address2: `
 	if (address1 === undefined || address2 === undefined) return false
 	return address1.toLowerCase() === address2.toLowerCase()
 }
+
+export function getSubstringAfterFirstPoint(input: string): string {
```

Comment:
"Point" may be a European thing?  This character is usually called a `Dot` or a `Period` I think?
```suggestion
export function getSubstringAfterFirstDot(input: string): string {
```

Reply:
```suggestion
export function getSubstringAfterFirstPeriod(input: string): string {
```